TrailBlaze/Networking/Interceptor.m

```diff
@@ -33,6 +33,11 @@ + (NSArray *) getRemainingRoutePoints: (NSArray *) routePoints runnerLocation: (
     return [routePoints subarrayWithRange:NSMakeRange(closestIndex, routePoints.count - 1)];
 }
 
-
++ (NSArray *) sortInAscendingOrder: (NSArray *) etaDifferences {
+    NSSortDescriptor *sortDescriptor;
+    sortDescriptor = [[NSSortDescriptor alloc] initWithKey:@"doubleValue" ascending:YES];
+    NSArray *sortDescriptors = [NSArray arrayWithObject:sortDescriptor];
```

Comment:
When declaring arrays, let's make sure to say what type the array is:
`NSArray<NSSortDescriptor *> *const sortDescriptors = ...`